server/hotrod/src/main/java/org/infinispan/server/hotrod/logging/Log.java

```diff
@@ -76,7 +76,7 @@
    @Message(value = "Factory '%s' not found in server", id = 6016)
    IllegalStateException missingKeyValueFilterConverterFactory(String name);
 
-   @Message(value = "Unauthorized '%s' operation", id = 6017)
+   @Message(value = "Authentication is required to perform the '%s' operation", id = 6017)
```

Comment:
```suggestion
   @Message(value = "Operation '%s' requires authentication", id = 6017)
```

Reply:
What about `Users must authenticate to perform the '%s' operation` ?

Reply:
I prefer Dan's